libkbfs/tlf_journal.go

```diff
@@ -809,6 +808,62 @@ func (j *tlfJournal) getJournalStatus() (TLFJournalStatus, error) {
 	}, nil
 }
 
+func (j *tlfJournal) getJournalStatus() (TLFJournalStatus, error) {
+	j.journalLock.RLock()
+	defer j.journalLock.RUnlock()
+	return j.getBaseJournalStatusLocked()
+}
+
+func (j *tlfJournal) getJournalStatusWithPaths(ctx context.Context,
+	config Config, mdOps journalMDOps, blocks *folderBlockOps) (
```

Comment:
passing around `folderBlockOps` makes me slightly uncomfortable, since it does so much and we only need it for getting paths. I think it's a bit cleaner to define a one-off interface somewhere with just the `SearchForPaths` method. You'll need to plumb the `NodeCache` along, or make a helper struct that pairs an FBO with a `NodeCache` and make that implement the interface.